### PR TITLE
 adapt cart page to design

### PR DESCRIPTION
**Pull request by dendoesit**
## Describe your changes
Bring cart page more to design.

### CODE REVIEW

File: apps/store/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "node --require ./server-preload.js ./node_modules/.bin/next start -p $PORT",
     "test": "jest --passWithNoTests --coverage",
-    "lint": "TIMING=1 next lint",
+    "lint": " next lint",
```
Comment (dendoesit) [inline apps/store/package.json:10]: @robinandeer I saw that you added the TIMING=1 in a previous PR. Is it mandatory ? It currently breaks the yarn dev script on windows.

Comment (robinandeer) [reply apps/store/package.json:10]: Sorry @dendoesit, I keep forgetting about that... It enables caching of the linting step so would be nice to keep but we should make it work cross-platform

Feel free to remove it in the mean time. Maybe we could try this: https://stackoverflow.com/questions/25112510/how-to-set-environment-variables-from-within-package-json/36373133#36373133

File: apps/store/src/components/CartPage/CartPage.tsx [hunk 2]
```diff
@@ -12,16 +13,36 @@ export const CartPage = () => {
...
+        <Button fullWidth={true}>
+          <Link href={PageLink.cartReview()}>Check Out</Link>
+        </Button>
+      </Footer>
+    </CartLayout>
   )
 }
+
+const CartLayout = styled.div(({ theme }) => ({
+  marginTop: theme.space[8],
```
Comment (robinandeer) [inline apps/store/src/components/CartPage/CartPage.tsx:33]: I suggest we avoid margins inside components -- better to move that concern to the parent scope and use something like `Space / SpaceFlex`

File: apps/store/src/components/LayoutWithMenu/LayoutWithMenu.tsx [hunk 3]
```diff
@@ -37,4 +38,5 @@ const FixedHeader = styled.header({
   top: 0,
   left: 0,
   right: 0,
+  backgroundColor: theme.colors.white,
```
Comment (robinandeer) [inline apps/store/src/components/LayoutWithMenu/LayoutWithMenu.tsx:41]: I think the idea is that content bleeds into the header -- like it just floats on top of everything else from looking at the design.

File: apps/store/src/components/CartCard/CartCard.tsx [hunk 4]
```diff
@@ -48,20 +49,20 @@ export type CartCardProps = {
...
         <IconElement />
         <HeaderElement>
           {title && <TitleElement>{title}</TitleElement>}
-          <RemoveButton>Remove</RemoveButton>
+          <RemoveButton onClick={() => handleRemoveProduct()}>Remove</RemoveButton>
         </HeaderElement>
-        {price &&
-          <ExtraElement>{currencyFormatter.format(price)}/mo.</ExtraElement>
-}
+        {price && <ExtraElement>{currencyFormatter.format(price)}/mo.</ExtraElement>}
```
Comment (robinandeer) [inline apps/store/src/components/CartCard/CartCard.tsx:65]: Is there a reason why we would display the `CartCard` without a price?

**Review by robinandeer — COMMENTED** [review]
Sorry @dendoesit, I keep forgetting about that... It enables caching of the linting step so would be nice to keep but we should make it work cross-platform

Feel free to remove it in the mean time. Maybe we could try this: https://stackoverflow.com/questions/25112510/how-to-set-environment-variables-from-within-package-json/36373133#36373133

File: apps/store/src/components/CartPage/CartPage.tsx (hunk 2 above)
Comment (dendoesit) [reply apps/store/src/components/CartPage/CartPage.tsx]: I was thinking that the cartLayout is the parent component of the CartPage. 
I really could not figure out how Space works. It seems to only add margin-top if there is already a previous element before.

File: apps/store/package.json (hunk 1 above)
Comment (dendoesit) [reply apps/store/package.json]: yeah that seems to do it. I added it to the PR.

File: apps/store/package.json [hunk 5]
```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "node --require ./server-preload.js ./node_modules/.bin/next start -p $PORT",
     "test": "jest --passWithNoTests --coverage",
-    "lint": "TIMING=1 next lint",
+    "lint": "export TIMING=1 || set TIMING=1 &&  next lint",
```
Comment (robinandeer) [inline apps/store/package.json:10]: that's awesome if this works

```suggestion
    "lint": "export TIMING=1 || set TIMING=1 && next lint",
```

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/components/CartCard/CartCard.tsx [hunk 6]
```diff
@@ -39,26 +39,30 @@ const RemoveButton = styled.button({
...
       <Content>
         <IconElement />
         <HeaderElement>
-          {title && <TitleElement>{title}</TitleElement>}
-          <RemoveButton>Remove</RemoveButton>
+          <TitleElement>{title}</TitleElement>
+          <RemoveButton onClick={() => handleRemoveProduct()}>Remove</RemoveButton>
         </HeaderElement>
-        {price && <ExtraElement>{currencyFormatter.format(price)}/mo.</ExtraElement>}
+        <ExtraElement>{currencyFormatter.format(price)}/mo.</ExtraElement>
```
Comment (robinandeer) [inline apps/store/src/components/CartCard/CartCard.tsx:65]: nit: could be easier to just pass price in as a string here (including the `/mo.` part

**Review by robinandeer — APPROVED** [review]

**Pull request merged**